lib/rmt/cli/base.rb

```diff
@@ -83,6 +85,17 @@ def subcommand_prefix
 
   private
 
+  def locked_method(&_block)
+    RMT::Lockfile.create_file
+    yield
+    RMT::Lockfile.remove_file
+  rescue RMT::Lockfile::ExecutionLockedError
+    puts 'Process is locked'
+  rescue StandardError
```

Comment:
Why only `StandardError`? Use `ensure` block instead to always execute the lock removal code.